packages/atlas/src/views/viewer/NftsView/NftsView.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState } from 'react'
+
+import { useNftsConnection } from '@/api/hooks'
+import { OwnedNftOrderByInput } from '@/api/queries'
+import { EmptyFallback } from '@/components/EmptyFallback'
+import { FiltersBar, useFiltersBar } from '@/components/FiltersBar'
+import { GridItem } from '@/components/LayoutGrid'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { SvgActionFilters } from '@/components/_icons'
+import { Select } from '@/components/_inputs/Select'
+import { NftTileViewer } from '@/components/_nft/NftTileViewer'
+import { VideoContentTemplate } from '@/components/_templates/VideoContentTemplate'
+import { useHeadTags } from '@/hooks/useHeadTags'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { useVideoGridRows } from '@/hooks/useVideoGridRows'
+import { SentryLogger } from '@/utils/logs'
+import { StyledPagination } from '@/views/studio/MyVideosView/MyVideos.styles'
+
+import { HeaderContainer, HeaderWrapper, StyledGrid } from './NftsView.styles'
+
+const SORT_OPTIONS = [
+  { name: 'newest', value: OwnedNftOrderByInput.CreatedAtDesc },
+  { name: 'oldest', value: OwnedNftOrderByInput.CreatedAtAsc },
+]
+
+export const NftsView: React.FC = () => {
+  const headTags = useHeadTags('Video NFTs')
+  const smMatch = useMediaMatch('sm')
+  const mdMatch = useMediaMatch('md')
+
+  const filtersBarLogic = useFiltersBar()
+  const {
+    filters: { setIsFiltersOpen, isFiltersOpen },
+    canClearFilters: { canClearAllFilters },
+    ownedNftWhereInput,
+    videoWhereInput,
+  } = filtersBarLogic
```

Comment:
Hmmm, now we could apply the same filters on MemberView and ChannelView. Right now we can only filter by status there. 
Just saying, let's do this as a follow-up maybe?

Reply:
Yeah, let's do that later